var/spack/repos/builtin/packages/py-httpx/package.py

```diff
@@ -11,37 +11,49 @@ class PyHttpx(PythonPackage):
     and async APIs, and support for both HTTP/1.1 and HTTP/2."""
 
     homepage = "https://github.com/encode/httpx"
-    pypi = "httpx/httpx-0.15.2.tar.gz"
+    pypi = "httpx/httpx-0.27.0.tar.gz"
 
     license("BSD-3-Clause")
 
+    version("0.27.2", sha256="f7c2be1d2f3c3c3160d441802406b206c2b76f5947b11115e6df10c6c65e66c2")
+    version("0.27.0", sha256="a0cb88a46f32dc874e04ee956e4c2764aba2aa228f650b06788ba6bda2962ab5")
     version("0.23.3", sha256="9818458eb565bb54898ccb9b8b251a28785dd4a55afbc23d0eb410754fe7d0f9")
     version("0.22.0", sha256="d8e778f76d9bbd46af49e7f062467e3157a5a3d2ae4876a4bbfd8a51ed9c9cb4")
     version("0.15.2", sha256="713a2deaf96d85bbd4a1fbdf0edb27d6b4ee2c9aaeda8433042367e4b9e1628d")
     version("0.11.1", sha256="7d2bfb726eeed717953d15dddb22da9c2fcf48a4d70ba1456aa0a7faeda33cf7")
 
     variant("http2", default=False, when="@0.15.2:", description="Enable http2 support")
 
+    depends_on("python@3.8", when="@0.27:", type=("build", "run"))
```

Comment:
Curious.  Why not include this after the `default_args` context below?

Reply:
Also, shouldn't this be `python@3.8:` (based on the two newer `pyproject.toml` files)?